tests/integration/test_kinesis.py

```diff
@@ -51,12 +53,9 @@ def assert_consumers(count):
             ConsumerName=consumer_name)['ConsumerDescription']
         self.assertEqual(consumer_description_by_arn, consumer_description_by_name)
 
-        # delete non-existing consumer and assert 1 consumer
-        client.deregister_stream_consumer(StreamARN=stream_arn, ConsumerName='_invalid_')
-        assert_consumers(1)
-
         # delete existing consumer and assert 0 remaining consumers
         client.deregister_stream_consumer(StreamARN=stream_arn, ConsumerName=consumer_name)
+        sleep(1)
```

Comment:
kinesis-mock applies latency to DeregisterStreamConsumer so a sleep is needed here.